Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginUIFactory.swift

```diff
@@ -78,6 +78,29 @@ struct SimplifiedLoginUIFactory {
             nc.pushViewController(webVC, animated: true)
         }
         
+        viewModel.onClickedContinueAsUser = {
+            assertionFetcher.fetchAssertion { result in
+                switch result {
+                case .success(let assertion):
+                    DispatchQueue.main.async {
+                        let session = client.createWebAuthenticationSession(withMFA: nil, loginHint: nil, assertion: assertion.assertion, extraScopeValues: [], completion: completion)
+                        viewModel.asWebAuthenticationSession = session
+                        
+                        if #available(iOS 13.0, *) {
+                            let context = ASWebAuthSessionContextProvider()
+                            session.presentationContextProvider = context //TODO: Perhaps should be passed in
```

Comment:
can we somehow test that it should be passed?

Reply:
Or this is the same situation as for regular login flow

Reply:
The regular login flow >13 passes the uiwindow in as a ContextProvider.
<13 does not rely on a passed in window/contextProvider.

I will create a new ticket for this. Having a uiwindow passed in could also alleviate, if we want the SDK to own the **presenting** of the SL-view,